source/jenkins.py

```diff
@@ -11,6 +11,7 @@
 import urllib
 import socket
 import json
+import os.path
```

Comment:
I don't think this import is needed since we're not using any of the `path` methods?